client/src/app/pages/sbom-scan/components/VulnerabilityTable.tsx

```diff
@@ -89,12 +89,15 @@ export const VulnerabilityTable: React.FC<IVulnerabilityTableProps> = ({
     isSortEnabled: true,
     sortableColumns: [
       "vulnerabilityId",
+      "severity",
       "affectedPackages",
       "published",
       "updated",
     ],
     getSortValues: (item) => ({
       vulnerabilityId: item.vulnerability.identifier,
+      severity:
+        Array.from(item.advisories.values())[0]?.opinionatedScore?.value ?? 0,
```

Comment:
Add sorting `Severity`. Only the first severity is taken for sorting operations.